source/deploy/server/preparations.rst

```diff
@@ -180,11 +177,100 @@ Certain proxy servers also provide a layer of caching which can make loading ima
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+| S3 Storage                                                  | FileSettings.AmazonS3Endpoint         | 443 (TLS)                         | TCP       | Outbound   |                                                               |
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+| SMTP                                                        | EmailSettings.SMTPPort                | 10025                             | TCP/UDP   | Outbound   |                                                               |
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+| Push Notifications                                          | EmailSettings.PushNotificationServer  | 443 (TLS)                         | TCP       | Outbound   |                                                               |
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+
+.. note::
+   
+   - All outbound ports may vary based on your specific configuration
+   - Mattermost can be configured to use an outbound proxy for any HTTP/HTTPS traffic (see below)
+   - Calls service may require additional ports 
+
+Outbound proxy configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your deployment requires using an outbound proxy, you can configure Mattermost using environment variables:
+
+1. Configure the proxy settings in your service configuration:
+
+   .. code-block:: text
+
+      Environment=HTTP_PROXY=http://proxy.example.com:3128
+      Environment=HTTPS_PROXY=https://proxy.example.com:3128
+      Environment=NO_PROXY=localhost,127.0.0.1,.internal.example.com
+
+2. For authenticated proxies, include credentials in the URL:
+
+   .. code-block:: text
+
+      Environment=HTTP_PROXY=http://username:password@proxy.example.com:3128
+      Environment=HTTPS_PROXY=https://username:password@proxy.example.com:3128
+
+3. The ``NO_PROXY`` variable can include:
+   - IP addresses (e.g., ``1.2.3.4``)
+   - CIDR ranges (e.g., ``1.2.3.4/8``)
+   - Domain names (e.g., ``example.com``)
+   - Subdomains (e.g., ``.example.com``)
+
+.. note::
+
+   When using an HTTPS proxy, ensure your Mattermost server has the proxy's root certificate configured to avoid connection issues.
+
+.. dropdown:: Example systemd Service Configuration
```

Comment:
The current Sphinx environment doesn't support the ``.. dropdown::`` directive. As a result, no content associated with this directive is displayed when the docs are generated.

Are you looking to collapse and expand content on the page or something else?

Reply:
Correct, I'd like this to be collapsed and optionally expanded if the user would like to, but different directives didn't work. Any recommendation?

Reply:
Yes. We'd need a Sphinx extension to support collapse/expand functionality in our environment. I have one in mind. Is this a must-have to get this PR in, or a nice-to-have we could iterate on over time?

Reply:
Definitely a nice-to-have - We can just make it a code block instead.